### PR TITLE
Add sentry for issue tracking

### PR DESCRIPTION
**Pull request by maxwofford**
Before this is merged @paked, could you set the SENTRY_DSN value on Heroku to be the DSN for this project: https://sentry.io/hack-club/website?

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -327,4 +330,4 @@ DEPENDENCIES
   validates_email_format_of
 
 BUNDLED WITH
-   1.12.5
+   1.14.3
```
Comment (paked) [inline Gemfile.lock]: Do we really want to update bundler?

**Comment by paked** [comment]
@MaxWofford: Could you fix the tests here? Then I'm happy to merge.

File: Gemfile.lock (hunk 1 above)
Comment (MaxWofford) [inline Gemfile.lock]: Just ran `docker-compose run web bundle install` for this. That field just marks the previously bundled with version -- it doesn't lock future bundles into using that version.

**Comment by MaxWofford** [comment]
@paked Should be fixed now!

**Pull request merged**